packages/node/src/utils/flush-chunks.ts

```diff
@@ -73,7 +73,15 @@ const processChunk = async (chunk, shareMap, hostStats) => {
   const [remote, request] = chunk.split('->');
 
   // If the remote is not defined in the global config, return
-  if (!globalThis.__remote_scope__._config[remote]) {
+  //@ts-ignore
```

Comment:
need to fix this, as invalidation of require cache doesnt work anymore due to it being tied to old node federation stuff